packages/expo-updates/ios/EXUpdates/EnabledAppController.swift

```diff
@@ -78,6 +78,7 @@ public class EnabledAppController: UpdatesStateChangeDelegate, InternalAppContro
     )
     startupProcedure.delegate = self
     stateMachine.queueExecution(stateMachineProcedure: startupProcedure)
+    shouldEmitJsEvents = true
```

Comment:
Looking at the PR that added it, I think the way this was designed was that it should only set by UpdatesModule `OnStartObserving`? cc @Kudo

Reply:
Aha that makes sense, I'll have a look and make that adjustment.

Reply:
It seems like what we really want is the change below -- however, if I don't set the flag to true in EnabledAppController, the JS API tests still fail. Testing locally to see if there is some timing issue.

```diff
--- packages/expo-updates/ios/EXUpdates/UpdatesModule.swift	2024-03-22 11:40:35
+++ packages/expo-updates/ios/EXUpdates/UpdatesModule.swift	2024-03-24 11:31:53
@@ -74,11 +74,11 @@
     }
 
     OnStartObserving {
-      AppController.shouldEmitJsEvents = true
+      AppController.sharedInstance.shouldEmitJsEvents = true
     }
 
     OnStopObserving {
-      AppController.shouldEmitJsEvents = false
+      AppController.sharedInstance.shouldEmitJsEvents = false
     }
 
     AsyncFunction("reload") { (promise: Promise) in
```

Reply:
Yes I think the right answer is to enabled the flag at the end of the startup procedure. Just added a commit to do that, it works locally.